modules/src/main/modules/search/elasticsearch/module.xml

```diff
@@ -8,17 +8,13 @@
 <module xmlns="urn:jboss:module:1.1" name="org.hibernate.search.elasticsearch" slot="${hibernate.search.version}">
     <resources>
         <resource-root path="hibernate-search-elasticsearch-${hibernate.search.version}.jar" />
-        <resource-root path="rest-${elasticsearchClientVersion}.jar" />
-        <resource-root path="sniffer-${elasticsearchClientVersion}.jar" />
-        <resource-root path="gson-${elasticsearchGsonVersion}.jar" />
```

Comment:
That's not how the modules work :)
Since *we* use it, it should be in a module *we depend on*.

The intent of the isolation is to avoid transitive dependencies: we do export all of `hibernate-search-elasticsearch` to the end user's classpath so that people can use our public API contained in there, but we don't want them to be able to use our `gson` jar as it might conflict with their own copy.